samples/dect/dect_phy/dect_shell/src/dect/common/dect_phy_api_scheduler.c

```diff
@@ -753,6 +790,7 @@ void dect_phy_api_scheduler_list_status_print(void)
 	}
 	SYS_DLIST_FOR_EACH_CONTAINER(&done_sheduled_list, iterator, dnode) {
 		done_count++;
+		printk("  Done list item phy handle: %d\n", iterator->phy_op_handle);
```

Comment:
Does this function use `printk()` on purpose, instead of `desh_print()`?

Reply:
Hmhh, maybe there was some legacy use from mdm callbacks.I do not see any reason currently why not desh_print(). I'll change this.